test/SourceKit/Sema/sema_diag_after_edit_fixit.swift

```diff
@@ -4,7 +4,7 @@ let a = 0; let b = 0 }; unresolved
 // Test that offsets of diagnostic ranges and fixits get updated correctly after the edit request
 
 // RUN: %sourcekitd-test -req=open %s -- %s == -req=print-diags %s \
-// RUN:    == -req=edit -pos=2:1 -replace="_" -length=5 %s -print-raw-response \
+// RUN:    == -req=edit -pos=2:1 -replace="_" -length=5 %s == -req=print-diags %s \
```

Comment:
This test was specifically checking the previous behavior, that 'stale' diagnostics still show up, after correctly adjusting their source locations to accommodate for an edit. With this change it essentially switches to testing that after the edit and the document-update notification that we get the 'sema' diagnostics with the expected locations.